fn_ansible_tower/README.md

```diff
@@ -29,11 +29,11 @@
   Specify all changes in this release. Do not remove the release 
   notes of a previous release
 -->
-| Release | Date | Notes |
-| ------- | ---- | ----- |
-| v1.0.2  | 2/2021 | Bug fix for paged results from templates, jobs, etc. |
-| v1.0.1  | 12/2020 | App Host support |
-| v1.0.0  | 2/2020 | Initial release |
+### v1.0.1
```

Comment:
I'd prefer to keep the table format for the README. And there should be an entry for v1.0.3

Reply:
okay, but have not modified this file, may be there file diff in appexchange and github